CHANGELOG.md

```diff
@@ -1,3 +1,142 @@
+# 1.6.10-beta01 (April 2024)
+
+_Changes since 1.6.2_
+
+## Highlights
+
+- Experimental multiplatform support of `Lifecycle` and `ViewModel`. See [documentation](https://www.jetbrains.com/help/kotlin-multiplatform-dev/compose-lifecycle.html) for details
+- Experimental multiplatform support of Jetpack Navigation. See [documentation](hhttps://www.jetbrains.com/help/kotlin-multiplatform-dev/compose-navigation-routing.html) for details or check out the [nav_cupcake project](https://github.com/MatkovIvan/nav_cupcake) which was converted from the [Navigate between screens with Compose](https://developer.android.com/codelabs/basic-android-kotlin-compose-navigation#2) Android codelab
```

Comment:
After we have an example and a doc, we should change the `nav_cupcake` link to it. To avoid dependencies on private repos